.crates2.json

```diff
@@ -1 +1 @@
-{"installs":{"galette 0.3.0 (git+https://github.com/rustbox/galette.git?rev=2ba00658608c4e4d1fd51d903a7799dcabe2f9cc#2ba00658608c4e4d1fd51d903a7799dcabe2f9cc)":{"version_req":null,"bins":["galette"],"features":[],"all_features":false,"no_default_features":false,"profile":"release","target":"x86_64-unknown-linux-gnu","rustc":"rustc 1.64.0 (a55dd71d5 2022-09-19)\nbinary: rustc\ncommit-hash: a55dd71d5fb0ec5a6a3a9e8c27b2127ba491ce52\ncommit-date: 2022-09-19\nhost: x86_64-unknown-linux-gnu\nrelease: 1.64.0\nLLVM version: 14.0.6\n"}}}
\ No newline at end of file
+{"installs":{"galette 0.3.0 (git+https://github.com/rustbox/galette.git?rev=2ba00658608c4e4d1fd51d903a7799dcabe2f9cc#2ba00658608c4e4d1fd51d903a7799dcabe2f9cc)":{"version_req":null,"bins":["galette"],"features":[],"all_features":false,"no_default_features":false,"profile":"release","target":"x86_64-unknown-linux-gnu","rustc":"rustc 1.62.1 (e092d0b6b 2022-07-16)\nbinary: rustc\ncommit-hash: e092d0b6b43f2de967af0887873151bb1c0b18d3\ncommit-date: 2022-07-16\nhost: x86_64-unknown-linux-gnu\nrelease: 1.62.1\nLLVM version: 14.0.5\n"}}}
```

Comment:
You gotta run `rustup update` my man: rustc from _July_?  And/or, we should drop a `.rust-toolchain.toml ` file in this repo too.